Pocket/pocket-ios
# IN-572: Fix animation for item cell when archiving

# IN-572
**Issue: ** Swiping actions were not animating the cells properly in My List and Archive.
**Fix: ** Remove explicit call to receive event on main thread. We want the snapshot to apply before completion of action (archive / moveToMyList). 

## Testing Steps
1. Navigate to My List
2. Swipe left on an Item to Archive and verify that item cell is animating properly
3. Navigate to Archive and swipe left 
4. Swipe left on an Item to Move to My List and verify that item cell is animating properly
